lib/ruby-stackoverflow/client/response_data.rb

```diff
@@ -1,14 +1,19 @@
 module RubyStackoverflow
   class Client
     class ResponseData
-      attr_reader :data, :has_more, :error
+      attr_reader :data, :has_more, :error, :raw_response, :raw_data, :quota_max, :quota_remaining, :backoff
```

Comment:
Wondering how we get that raw_response and raw_data
I can see  :quota_max, :quota_remaining, :backoff here https://api.stackexchange.com/docs/wrapper